common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
_.isNumber returns true on NaN, which enrages me. :)  _.isFinite is the easiest check I found to ensure that the value is an actual "number" in any real sense of the term.

> Also, is there any case in which the value is set, but it is not a number? 

At the moment, no, not that I'm aware of, but code changes in future might let that happen. _.isFinite protects against that. Paranoia I suppose.

I was basing it on this code and comment (and I'll be modifying that code similarly to "check and clean timezoneOffset"):
````
function sanitizeOptions (o) {
  let ref = Number(o.dayStart || 0);
  let dayStart = !_.isNaN(ref) && ref >= 0 && ref <= 24 ? ref : 0;
  let timezoneOffset = o.timezoneOffset ? Number(o.timezoneOffset) : Number(moment().zone());
  // TODO: check and clean timezoneOffset as for dayStart (e.g., might not be a number)
````